### PR TITLE
Price order change & design updates.

### PR DESCRIPTION
**Pull request by remybach**
The height of the "top" bit of the cards needs to be the same when there are multiple of them next to each other. There isn't really a way to do it in a way that still supports IE9 (no css grid support) that doesn't involve making the markup really weird for the use cases where only a single card is used.

Happy to get feedback on any better ideas as this was a kind of "we have no other choice" approach.

ft-o-subs-card

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ The markup has been rearranged, and some classes have been removed.
 -				<img class="o-subs-card__copy-image" src='/demos/src/demo-placeholder.png' alt='photo of FT subscription options'>
```
Comment (remybach) [inline README.md:105]: Lol @ github marking this up like this. It's actually a diff in the readme.

File: src/scss/_mixins.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 	// https://github.com/philipwalton/flexbugs#flexbug-17 (for suport in ie 9/10)
 	@include oGridRespondTo(L) {
-		flex: 0 1 25%;
+		flex: 0 1 calc(25% - 12px);
```
Comment (gvonkoss) [inline src/scss/_mixins.scss]: If this is a magic number, could we have a variable that explains it in its place?

**Review by gvonkoss — APPROVED** [review]
There is a minor pedantic comment, but other than that :+1:

File: src/scss/_mixins.scss (hunk 2 above)
Comment (remybach) [reply src/scss/_mixins.scss]: Ah, yep... it's based on the 6px padding each side. Will add a comment stating that.

Comment (gvonkoss) [reply src/scss/_mixins.scss]: 👍

**Pull request merged**